src/main/java/seedu/address/model/module/ModuleCode.java

```diff
@@ -38,6 +71,23 @@ public static boolean isValidModuleCode(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    /**
+     * Returns the ArrayList of tutorial classes under this module.
+     *
+     * @return the ArrayList of tutorial classes.
+     */
+    public ArrayList<TutorialClass> getTutorialClasses() {
+        return tutorialClasses;
+    }
+
+    public String getTutorialClassesNames() {
+        StringBuilder output = new StringBuilder();
+        for (int i = 0; i < tutorialClasses.size(); i++) {
+            output.append(tutorialClasses.get(i).toString());
```

Comment:
Will this result in the correct format for the output? I believe you need to add the new line character to print each tutorial classes nicely, else, do you want to consider some separators inbetween each tutorial class?